src/ray/common/scheduling/cluster_resource_scheduler.cc

```diff
@@ -559,17 +603,24 @@ bool ClusterResourceScheduler::SubtractNodeAvailableResources(
     return false;
   }
 
+  FixedPoint zero(0.);
+
   for (size_t i = 0; i < PredefinedResources_MAX; i++) {
     resources.predefined_resources[i].available =
-        std::max(0., resources.predefined_resources[i].available -
-                         task_req.predefined_resources[i].demand);
+        resources.predefined_resources[i].available -
```

Comment:
I think `std::max(FixedPoint(0), resources.predefined_resources[i].available - task_req.predefined_resources[i].demand);` would be clearer here.

Reply:
done